sample-cl-rest-config.json

```diff
@@ -1,8 +1,9 @@
 {
     "PORT": 3001,
     "DOCPORT": 4001,
-    "PROTOCOL": "https",
```

Comment:
Hi @bnonni I would like to keep the default to https. I believe if the user wants to choose http they should consciously take that decision.

Can you please remove this change from the PR? Thanks.

Reply:
@saubyk understood, changed this back.